draft-ietf-rats-eat.md

```diff
@@ -564,6 +570,9 @@ this are:
   may find they can optimize their process by switching from type 0x01
   to type 0x02 or vice versa.
 
+The type byte is needed to distinguish UEIDs of different types that by chance have the same identifier value, but do not identify the same entity.
+The type byte MUST be treated as part of the opaque UEID and MUST not be used to make use of the internal structure of the UEID.
```

Comment:
What do the EUI and IMEI types bring to the table?  
Why not just have a 32-byte RAND UEID and be done with it?

Reply:
EUI and IMEI are much shorter, they may be ready-to-hand and and a random number generator might not be.